socialhome/streams/templatetags/slurp_json_context.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context(context):
```

Comment:
This template tag should have a few very small basic tests if it is permanent code.

Tests should be placed by (our) convention in `streams/tests/test_templatetags.py`. See test notes here: http://socialhome.readthedocs.io/en/latest/contributing.html#tests .. just a few simple tests to ensure basic behaviour - you don't need to create more complex data.